js/apps/admin-ui/src/identity-providers/IdentityProvidersSection.tsx

```diff
@@ -42,6 +42,7 @@ import { useFetch } from "../utils/useFetch";
 import { ManageOrderDialog } from "./ManageOrderDialog";
 import { toIdentityProvider } from "./routes/IdentityProvider";
 import { toIdentityProviderCreate } from "./routes/IdentityProviderCreate";
+import { IdentityProvidersQuery } from "libs/keycloak-admin-client/lib/resources/identityProviders";
```

Comment:
This should be imported using the library by name, rather than providing a path.
```suggestion
import { IdentityProvidersQuery } from "@keycloak/keycloak-admin-client/lib/keycloak-admin-client/lib/resources/identityProviders";
```